corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -375,7 +386,31 @@ hqDefine("geospatial/js/geospatial_map", [
         var missingGPSModel = function(cases) {
             this.casesWithoutGPS = ko.observable(cases);
         };
-        var missingGPSModelInstance = new missingGPSModel([]);
+        function loadCases(caseData) {
+            map.removeCaseMarkers();
+            var casesWithGPS = caseData.filter(function (item) {
+                return item[1] !== null;
```

Comment:
I see there's a lot of indexing happening in this function. What do you think about creating some sort of mapping that makes reading this more explicit? For example,
```
const caseItemMapping = {
    ID: 0,
    COORDINATES: 1,
    LINK: 2
}

_.object(_.map(casesWithGPS, function (item) {
    var caseId = item[caseItemMapping.ID];
    var coordinates = item[caseItemMapping.COORDINATES];
    ....
}
```
Totally up to you if you want to do that, but I would personally prefer that. Other devs can also chime in.

Reply:
It does look like this code was simply moved from a different place, so its okay to not do it as a part of this PR.
There is already a lot happening in this PR so good to take this improvement as a followup, if you do.

Reply:
That would be nice yes